platform/ios/app/Assets.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -1,6 +1,16 @@
 {
   "images" : [
     {
+      "idiom" : "iphone",
```

Comment:
These changes look unrelated. If they’re needed, would you mind splitting them out into a separate PR? Thanks.